apps/parsley/src/hooks/useSections/utils.ts

```diff
@@ -18,13 +19,15 @@ interface SectionLineMetadata {
  * or null if the input line does not indicate the start or end of a section.
  */
 export const processLine = (str: string): SectionLineMetadata | null => {
-  const regex = /(Running|Finished) command '([^']+)' in function '([^']+)'.*/;
+  const regex =
+    /(Running|Finished) command '([^']+)' in function '([^']+)' \(step ([^)]+)\)[^.]*\./;
```

Comment:
I would also maybe suggest adding some tests to ensure the regex works for decimals, sometimes the steps are like `(step 1.2 of 2)`. Here's an [example](https://parsley.mongodb.com/evergreen/evergreen_ui_spruce_e2e_patch_b7ebed3a2207b930cc5186f93dce42226a03a43a_668be525468e5600073d80eb_24_07_08_13_09_58/0/task?bookmarks=0,2475&shareLine=19) of that